src/app/static/src/tests/root/getters.test.ts

```diff
@@ -0,0 +1,46 @@
+import {mockModelCalibrateState, mockModelOptionsState, mockModelRunState, mockRootState} from "../mocks";
+import {getters} from "../../app/store/root/getters";
+import {Warning} from "../../app/generated";
+import {STEPS} from "../../app/root";
+
+describe(`root getters`, () => {
+
+    const warnings: Warning[] = [
```

Comment:
I think these would be slightly better tests if the mock state modules did not have identical warning arrays, so that we could test that warnings from each of the origins is being sourced correctly. 

Could we also include tests for warnings to be displayed on a step other than the origin e.g. show model options warning on model_fit or on review_output.